### ISSUE TITLE
[FE] Document Testing Steps for CI and PCI On Stage

### ISSUE THREAD

**Issue by julieadair1** [department-of-veterans-affairs/va.gov-team#36748]
## How to configure this issue

- [ ] Attached to an Epic (what body of work is this a part of?)
- [ ] Correctly Labeled
- [ ] Fill in User Stories
- [ ] Fill in Documentation
- [ ] Fill in Tasks
- [ ] Fill in Acceptance Criteria

---

## User Story

As an internal dev user I need to know how to set up appointments for pre check in and day of check in in the staging environment so that I can complete testing to ensure quality. 

- (%FILL IN%)
- (%FILL IN%)

## Related Documentation

> these should be links to feature docs

- [Sketch](https://www.sketch.com/s/e79a827e-42cf-4a82-b554-874c75b5c70e)
- [Product Docs](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/health-care/checkin)
- [Feature Doc](%FILL IN%)

## Description

> this should be an understandable description of the ticket. This description should have enough context meaning that non-technical team members should get the idea.

## Tasks

1. Document testing steps and post to github. Include instructions on getting started with the test harness. 

- [ ] %Add any tasks%
- [ ] Create or update unit tests
- [ ] Create or update e2e tests
- [ ] Update relevant documentations
  - [ ] README
  - [ ] Use Cases

## Acceptance Criteria

1. Ensure that all team members have been offered an up to date copy of the test harness via keybase. 

- [ ] %Add any additional Acceptance Criteria%
- [ ] Passes Accessibility checks using axe
- [ ] Deployed to staging.
- [ ] Designer has signed off on the feature
- [ ] Product team has signed off on the  feature

**Comment by brianseek** [comment]
@mdewey @dcgoodwin2112 @acrollet - I noticed that we have most of the testing endpoints and steps outlined here: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/checkin/engineering/qa/test-data-setup.md

I made a few minor updates to it. It is not quite a step by step guide for pre-check-in and day of check-in but all the info is there. Let me know if you still think we need something more explicit or if there is any other details you would add in.
